settings/authentication-personalization/authentication-setup/jwt.mdx

```diff
@@ -100,3 +100,20 @@ async def return_mintlify_auth_status(current_user):
   return RedirectResponse(url=f'https://docs.foo.com/login/jwt-callback#{jwt_token}', status_code=302)
 ```
 </CodeGroup>
+
+## Redirecting Unauthenticated Users
+
+When an unauthenticated user tries to access a specific page, Mintlify preserves their intended destination through a redirect flow:
+
+1. The user attempts to visit a certain page (e.g., `/quickstart`)
+
+2. Mintlify redirects them to your login URL and adds the original destination as a `redirect` query parameter
```

Comment:
nit: maybe explicitly mention that Mintlify includes a relative redirect parameter, not an absolute URL? It should be obvious but you never know

Reply:
ah yeah great call